src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -164,6 +253,49 @@ class LicenceConditionService(
         }
       }
     }
+
+    /*
+    Using this function, an additional condition is either a new condition or to be removed. Updates to existing
+    additional conditions only involve updating the data associated with the condition using the updateAdditionalConditionData
+    function
```

Comment:
I don't _think_ this comment is necessarily true, but I don't think it has an impact on the logic used below.
I _think_ `updatedAdditionalConditions` can include conditions that haven't changed themselves, but will now appear in a different order on the licence due to other conditions being added or removed. No biggie and as you're sticking to auditing the `newAdditionalConditions` and `removedAdditionalConditions` I think it's fine (imo we don't need to audit sequence changes as there's no relevant user-input for those)

Reply:
if not true then it'd be worth removing the comment

Reply:
Whoops, I forgot to take that out - used the comments to aid me when working through the code - I will update the comment (and place in my own notes) and remove it from the code.